duo-universal-sdk/src/main/java/com/duosecurity/model/AccessDevice.java

```diff
@@ -1,13 +1,39 @@
 package com.duosecurity.model;
 
 import java.io.Serializable;
-import lombok.Data;
 
-@Data
 public class AccessDevice implements Serializable {
   private static final long serialVersionUID = -1130960392429229150L;
 
   private String hostname;
   private String ip;
   private Location location;
+
```

Comment:
How did you ensure that the getters you have created have the same names as the ones lombok was making?

Reply:
I delomboked the getters (i.e. used Lombok to generate the code). However, I realized that for comparability, we need more than just getters and setters. Therefore, I updated the models to include additional generated methods and constructors.